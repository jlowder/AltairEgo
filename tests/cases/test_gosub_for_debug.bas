10 FOR I=1 TO 3
20 PRINT "TRIAL #";I
30 GOSUB 100
40 PRINT "BACK FROM GOSUB"
50 NEXT I
60 PRINT "DONE"
70 END
100 PRINT "IN SUBROUTINE"
110 RETURN
