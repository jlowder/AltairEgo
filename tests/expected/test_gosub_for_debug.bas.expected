Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
TRIAL # 1
IN SUBROUTINE
BACK FROM GOSUB
TRIAL # 2
IN SUBROUTINE
BACK FROM GOSUB
TRIAL # 3
IN SUBROUTINE
BACK FROM GOSUB
DONE
OK
